portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,17 +53,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       comments.add(comment);
     }
 
-    Gson gson = new Gson();
+    Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    String rawString = gson.toJson(comments);
+    String finalString = rawString.replace("[\n", "").replace("]", "").replace(",", "").replace("{\n", "").replace("}", "");
 
-    response.setContentType("application/json;");
-    response.getWriter().println(gson.toJson(comments));
+    response.setContentType("text/html;");
+    response.getWriter().println(finalString);
   }
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Date time = new Date();
     String username = getParameter(request, "username", "") + ": ";
+    if (username.equals(": ")){
```

Comment:
I wonder if instead of storing the colon as part of the username, it'd be reasonable to add the colon client-side later? e.g. when you display the comments on the page, stringify it by going username + ": " + comment

Could be useful in the future if there's ever a time when you're displaying it in a way that doesn't require a colon. Anyway this is a non-urgent issue, so approving :)